pages/access-list.js

```diff
@@ -43,17 +39,22 @@ const AccessListPage = ({ first480 }) => {
         />
         <Hero totalReserved={first480.length} />
         <Header
-          count={first480.length}
-          maximum={100}
+          count={alphaList.length}
+          maximum={ALPHA_LIST_MAXIMUM}
           title="Alpha List"
           variant="primary"
         />
-        <Header count={first480.length} title="Waitlist" variant="secondary" />
-        <AccessListsContianer>
-          <AccessListMemberGrid members={first30} size="large" />
-          <AccessListMemberGrid members={second100} size="medium" />
-          <AccessListMemberGrid members={third350} size="small" />
-        </AccessListsContianer>
+        <AccessListContainer members={alphaList} variant="primary" />
+        {waitList.length ? (
```

Comment:
Again here, sorry if it's just pointing something that won't be updated maybe but in the case that we want to it will be helpful :D

Reply:
All good! Great point on the ternaries. I’m basically OK with them for simple uses, especially the `: null` version, as it reduces the boilerplate needed to render (again when simple). Multi-line I’m OK with as well, as long as the `()` and indents make it readable, which prettier thankfully enforces.